PassleSync.Website/App_Plugins/PassleSync/backoffice/passleSync/components/organisms/postsTable/postsTable.controller.js

```diff
@@ -1,270 +1,31 @@
-                vm.isSelectedAll = false;
-                vm.selectedCount = 0;
-
-                UpdatePagination();
-                UpdateItemsOnShow();
-
-                vm.isLoading = false;
-            }, (error) => {
-                console.error(error);
-                notificationsService.error("Error", error);
-                vm.isLoading = false;
-            });
-        }
-        onload();
-
-        vm.update = function () {
-            vm.isUpdating = true;
-
-            passlePostsResource.updateAll().then((response) => {
-                vm.posts = response.Posts.map((post) => getPostDataObject(post));
-
-                vm.syncedCount = vm.posts.filter((post) => post.synced).length;
-                vm.unsyncedCount = vm.posts.length - vm.syncedCount;
-                vm.isSelectedAll = false;
-                vm.selectedCount = 0;
-
-                UpdatePagination();
-                UpdateItemsOnShow();
-
-                vm.isUpdating = false;
-
-                syncTree();
-            }, (error) => {
-                console.error(error);
-                notificationsService.error("Error", error);
-
-                vm.isUpdating = false;
-            });
-        }
-
-        vm.sync = function () {
-            vm.isUpdating = true;
-
-            let syncProm;
-            let shortcodes = [];
-
-            if (vm.selectedCount == 0) {
-                syncProm = passlePostsResource.syncAll();
-            } else if (vm.selectedCount == 1) {
-                shortcodes = vm.posts.filter((post) => post.selected).map((post) => post.shortcode);
-                syncProm = passlePostsResource.syncOne(shortcodes);
-            } else {
-                shortcodes = vm.posts.filter((post) => post.selected).map((post) => post.shortcode);
-                syncProm = passlePostsResource.syncMany(shortcodes);
-            }
-
-            syncProm.then((response) => {
-                vm.posts.forEach((post, ii) => {
-                    let matchingPosts = response.Posts.filter(x => x.Shortcode === post.shortcode);
-                    if (matchingPosts.length > 0) {
-                        vm.posts[ii] = getPostDataObject(matchingPosts[0]);
-                    }
+        vm.entityInfo = {
+            nameSingle: "Post",
+            namePlural: "Posts",
+            getRowData: function getRowData(post, deletedOverride = false) {
+                // Convert the returned model to the format the table needs
+                // The override is used when deleting data:
+                // - we return the last state of the content before it was deleted, so we need to override the synced and url values
+
+                return Object.assign({}, {
+                    "name": post.Title,
+                    "excerpt": post.Excerpt,
+                    "editPath": (post.Synced && !deletedOverride) ? ("/content/content/edit/" + post.Id) : post.PostUrl,
```

Comment:
Since we don't allow editing Passle content within the CMS, I think it would make more sense to always return the `post.PostUrl` here. I believe this is what we do in the WordPress plugin. Based on this, it would also make sense to rename `editPath` to something like `url`.